src/test/testHooks.ts

```diff
@@ -1,20 +1,50 @@
 import { Context } from 'mocha';
-import { Telemetry } from '../platform/common/constants';
-import { sendTelemetryEvent } from '../telemetry';
+import { AppinsightsKey, JVSC_EXTENSION_ID, Telemetry } from '../platform/common/constants';
+import TelemetryReporter from '@vscode/extension-telemetry/lib/telemetryReporter';
 import { IS_CI_SERVER } from './ciConstants.node';
+import { extensions } from 'vscode';
+import { sleep } from '../platform/common/utils/async';
+
+let telemetryReporter: TelemetryReporter;
 
 export const rootHooks = {
```

Comment:
types to the rescue..

```suggestion
export const rootHooks: : Mocha.RootHookObject = {
```

Reply:
This should pick up other typos in the future.